truffle-config.js

```diff
@@ -0,0 +1,2 @@
+// Note: this config is only here as a matter of convenience to allow all truffle commands that can be run from core work at root as well.
+module.exports = require("@umaprotocol/common").getTruffleConfig("./core");
```

Comment:
this is the only instance where you specify a direct path because it will always be executed from the `root`, right? could you have used the `const path = require("path");` syntax here otherwise?

Reply:
This is a good point. This is because root is never going to be a public package and it technically doesn't (and maybe can't?) depend on one of its sub-packages. I think that syntax might work, but it really shouldn't because `@umaprotocol/core` isn't a dependency of root.